samples/quick-start/docker-compose.yml

```diff
@@ -1,9 +1,11 @@
-version: '3'
+version: '3.5'
 services:
   eventstore:
-    image: eventstore/eventstore:20.10.0-buster-slim
+    image: eventstore/eventstore:latest
     environment:
-      - EVENTSTORE_INSECURE=true
-      - EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP=true
+      EVENTSTORE_INSECURE: true
+      EVENTSTORE_MEM_DB: false
```

Comment:
memdb is false by default (i don't mind if you want to keep this line anyway though)

Reply:
added it to make it explicit but can easily remove it.